.github/workflows/ci.yml

```diff
@@ -3,6 +3,8 @@ name: GitHub Actions
 
 on:
   pull_request:
+      paths-ignore:
```

Comment:
The intention is not to run the full CI when the custom tests CI is edited in an ongoing PR, but it is not currently working due to https://github.com/actions/runner/issues/2324.